dev/update_ml_package_versions.py

```diff
@@ -90,18 +88,6 @@ def update_max_version(src, key, new_max_version, category):
     return re.sub(pattern, rf'\g<1>"{new_max_version}"', src, flags=re.DOTALL)
 
 
-def parse_args(args):
-    parser = argparse.ArgumentParser()
-    parser.add_argument(
-        "-p",
-        "--path",
-        help="Path to the ML package versions yaml (default: mlflow/ml-package-versions.yml)",
-        default="mlflow/ml-package-versions.yml",
-        required=False,
-    )
-    return parser.parse_args(args)
```

Comment:
This argument was added to avoid modifying `mlflow/ml-package-versions.yml` in the repository. We no longer need this argument because we have the `change_working_directory` fixture.